Flaky test: TestDaemonRestartRestoreBridgeNetwork

### Description

```
=== Failed
=== FAIL: amd64.integration-cli TestDockerDaemonSuite/TestDaemonRestartRestoreBridgeNetwork (1.99s)
    docker_cli_network_unix_test.go:1695: 80 port is allocated to old running container, it should failed on allocating to new container
    check_test.go:554: [df88f57742269] daemon is not started
    --- FAIL: TestDockerDaemonSuite/TestDaemonRestartRestoreBridgeNetwork (1.99s)

=== FAIL: amd64.integration-cli TestDockerDaemonSuite (240.90s)
```

<!-- Failed to upload "logs_39783360412.zip" -->


### docker version

145834f2785d8385ee47255fe3db61643eedf47e

--- a/integration-cli/docker_cli_network_unix_test.go
+++ b/integration-cli/docker_cli_network_unix_test.go
@@ -1691,8 +1691,8 @@ func (s *DockerDaemonSuite) TestDaemonRestartRestoreBridgeNetwork(t *testing.T) 
 
 	// start a new container, trying to publish port 80:80 should fail
 	out, err := s.d.Cmd("run", "-p", "80:80", "-d", "busybox", "top")
-	if err == nil || !strings.Contains(out, "Bind for 0.0.0.0:80 failed: port is already allocated") {
-		t.Fatalf("80 port is allocated to old running container, it should failed on allocating to new container")
+	if err == nil || !strings.Contains(out, "port is already allocated") {
+		t.Fatalf("Expected 'port is already allocated', got err:%v out:%s", err, out)
 	}
 
 	// kill old running container and try to allocate again

